packages/css/src/stories/components/buttons/buttons.stories.js

```diff
@@ -6,6 +6,8 @@ import secondaryReversed from './examples/secondary-reversed.html';
 import ghost from './examples/ghost.html';
 import conversion from './examples/conversion.html';
 
+import '~/src/index.css';
```

Comment:
I think this should be put in the storybook preview.js file ... ? it's more maintainable...

Reply:
Do you have an example ?

Reply:
```
import '../src/index.css';

export const parameters = {
  actions: { argTypesRegex: '^on[A-Z].*' },
};

```